JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -258,6 +277,10 @@ module Microsoft.ApplicationInsights {
          * helper method to access userId and sessionId cookie
          */
         public static getCookie(name) {
+            if (!Util.canUseCookies()) {
```

Comment:
Out of curiosity, can a flag be set at startup rather than making the method call each time?
So "this.canUseCookies" instead of "Util.canUseCookies()"?